doc/reference/configuration/configuration_reference.rst

```diff
@@ -2439,7 +2505,11 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.pid_file
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
     Store the process id in this file.
+    Can be relative to :ref:`process.work_dir <configuration_reference_process_work_dir>`.
+    A typical value is “:file:`tarantool.pid`”.
```

Comment:
I think that a typical value is the default (`var/run/{{ instance_name }}/tarantool.pid`) and we don't need to mention it specifically.